frontend/src/components/Quiz/Category.jsx

```
@@ -0,0 +1,99 @@
+  }
+
+  return (
+    <div className="text-center">
+      <h2 className="text-2xl font-semibold mb-8">Select a Quiz Category</h2>
+      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6">
+        {categories.map(category => (
+          <div 
+            key={category.id} 
+            className="bg-white rounded-lg shadow-md hover:shadow-lg transition-shadow duration-300 overflow-hidden cursor-pointer"
+            onClick={() => handleCategorySelect(category.id)}
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)